### DISCUSSION TITLE
Some Posts Not Showing On Home

### DISCUSSION THREAD

**Discussion by jbassetti** [adityatelange/hugo-PaperMod#1088] [General]
Hi, new Hugo user here. So far, loving HUGO and this theme.

But I am encountering an issue where certain posts are not showing up on the homepage (Home-Info Mode).

In "Content" I have several Folders for categories. Here is an example of my content folder structure:

> Content
>>Folder-Category-1
>>>First-Post-Folder
>>>>index.md

> Content
>>Folder-Category-1
>>>Second-Post-Folder
>>>>index.md

> Content
>>Folder-Category-2
>>>First-Post-Folder
>>>>index.md

> Content
>>About
>>>index.md

> Content
>>Archives
>>>index.md

> Content
>>Search
>>>index.md

On my homepage, only the content in "Folder-Category-1" appears under the social media links. 

The posts in "Folder-Category-2" and even those in the “Content" root aren’t shown. 

I would like the posts in the folders to show. Can anyone help me with this?

**Comment by adityatelange** [comment]
https://github.com/adityatelange/hugo-PaperMod/wiki/FAQs#posts-from-only-one-foldersection-visible-on-home-page

**Reply by jbassetti** [reply]
When I place the code with the appropriate folders in my config, this overrides the Home-Info Mode text and icons, which I don't want.

**Reply by jbassetti** [reply]
Got it to work by deleting params and placing it under HomeInfoParams. Thank you!
